src/locale/en.json

```diff
@@ -18,6 +18,9 @@
       "hashtag-cancel-rent": "Rent Strike resources for #CancelRent."
     }
   },
+  "searchbar": {
+    "default-value": "Enter nation, state, province, city or zipcode"
```

Comment:
Per our conversation with @bretthalperin shouldn't this be `Enter location...`?

Reply:
@timhitchins Yes, I was thinking we change it to "Search location"